components/object.js

```diff
@@ -2,48 +2,14 @@
 
 var _ = require('underscore');
 
-var promiseOutput = require('../src/promise-output');
-var componentFactory = require('../src/noflo-component-factory');
+var wrapper = require('../src/javascript-wrapper.js');
```

Comment:
I am curious why you felt that we needed to make an object updater.  An updater that wishes to pass an object can already define it and send it with a VNI.  Why was this primarily so your subgraphs could create objects with a VNI associated?  Other?